stories/setup/webpack.config.babel.js

```diff
@@ -1,57 +1,16 @@
-import 'core/polyfill';
-
 import path from 'path';
 
-import config from 'config';
-import autoprefixer from 'autoprefixer';
-import webpack from 'webpack';
-
-import { getClientConfig } from 'core/utils';
+import { getPlugins, getRules } from '../../webpack-common';
 
 module.exports = {
   module: {
-    rules: [
-      {
-        test: /\.svg$/,
-        use: [{ loader: 'svg-url-loader', options: { limit: 10000 } }],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader', options: { importLoaders: 2 } },
-          {
-            loader: 'postcss-loader',
-            options: {
-              plugins: [
-                autoprefixer({
-                  browsers: ['last 2 versions'],
-                  grid: false,
-                }),
-              ],
-            },
-          },
-          { loader: 'sass-loader', options: { outputStyle: 'expanded' } },
-        ],
-      },
-    ],
+    rules: [...getRules({ bundleStylesWithJs: true })],
   },
-  // The following plugins are needed to help handle
-  // server-side imported components such as core/logger.
-  // Is this the best way??
-  plugins: [
-    new webpack.DefinePlugin({
-      CLIENT_CONFIG: JSON.stringify(getClientConfig(config)),
-    }),
-
-    new webpack.NormalModuleReplacementPlugin(
-      /config$/,
-      'core/client/config.js',
-    ),
-  ],
+  plugins: [...getPlugins({ includeCircularDependencyPlugin: false })],
   resolve: {
     alias: {
       normalize: 'normalize.css/normalize.css',
+      tests: path.resolve('./tests'),
```

Comment:
This is not working as expected. If I leave this out, the component can't find the module. If I add it I still need to do ../../test etc..